lib/vim_ahk.ahk

```diff
@@ -69,6 +71,7 @@ class VimAhk{
     GroupAdd, VimCursorSameAfterSelect, ahk_exe explorer.exe ; Explorer
 
     ; Configuration values for Read/Write ini
+    ; setting, default, val, description, info
```

Comment:
Not needed.

Reply:
Not needed, but very useful. Saves you having to look up the function definition when adding new entries.

Reply:
If you have an editor to follow the functions, it is better and easier to check the function itself.
If it is necessary, it is always needed to write arguments of functions for each usage...

Reply:
Fair point, although I'd argue this is a special case, since it's a function used a lot here only, with very specific fields, yet the function itself is defined in another file.

I assume your first point is suggesting a modern IDE should show the signature anyway, and I agree for most other languages. However, AHK has no such features, so in this case I think it's still worth leaving in.

If you still disagree, I will remove it.